app/demo_data/fake_student.rb

```diff
@@ -246,7 +246,7 @@ def add_discipline_incidents
   def add_deprecated_interventions
     15.in(100) do
       generator = FakeInterventionGenerator.new(@student)
-      intervention_count = Rubystats::NormalDistribution.new(3, 6).rng.round
+      intervention_count = [3, 6].sample
```

Comment:
maybe `intervention_count = if rand() > 0.5 then 0 else (1..6).sample end` to keep more zero points, and less overall records so we don't all a lot more to the db?